src/constructs/acm/certificate.ts

```diff
@@ -0,0 +1,87 @@
+import { Certificate, CertificateValidation } from "@aws-cdk/aws-certificatemanager";
+import type { CertificateProps } from "@aws-cdk/aws-certificatemanager/lib/certificate";
+import { HostedZone } from "@aws-cdk/aws-route53";
+import { RemovalPolicy } from "@aws-cdk/core";
+import { Stage } from "../../constants";
+import type { GuStack } from "../core";
+import { AppIdentity } from "../core/identity";
+import { GuMigratingResource } from "../core/migrating";
+import type { GuStatefulConstruct } from "../core/migrating";
+
+export type GuCertificateProps = Record<Stage, GuDnsValidatedCertificateProps> & GuMigratingResource & AppIdentity;
+
+export interface GuDnsValidatedCertificateProps {
+  domainName: string;
+  hostedZoneId?: string;
+}
+
+/**
+ * Construct which creates an ACM Certificate.
```

Comment:
minor: Should we make it explicit that this is a DNS validated cert? Either by mentioning it in this comment or in the naming of the construct?

Reply:
Good idea, I've [updated the docs](https://github.com/guardian/cdk/pull/390/commits/17a65a5c711727d39a613e3b8bab64e9d3024758).